setup.cfg

```diff
@@ -9,5 +9,5 @@ tests = tests
 [flake8]
 ignore=E501,F401,F403,F405,I201
 exclude=build,dist,docs,venv,.tox,.eggs,tests
-max-complexity=42
+max-complexity=45
```

Comment:
Finally ran across a comment to the bug this PR corrects in my work codebase; looked, cool, 2016, it's probably fixed.

Then I see this as part of the correction. What. <!-- My own DAO: 5 paths exceed 1, worst-case is a single path at 3. -->